operator/controllers/loki/lokistack_controller.go

```diff
@@ -216,7 +223,7 @@ func (r *LokiStackReconciler) buildController(bld k8s.Builder) error {
 	if r.FeatureGates.OpenShift.Enabled {
 		bld = bld.
 			Owns(&routev1.Route{}, updateOrDeleteOnlyPred).
-			Watches(&cloudcredentialv1.CredentialsRequest{}, r.enqueueForCredentialsRequest(), updateOrDeleteOnlyPred)
+			Owns(&cloudcredentialv1.CredentialsRequest{}, updateOrDeleteOnlyPred)
```

Comment:
I thought about making it optional as well, but then did not because is already wasn't. :slightly_smiling_face: 

A thought crossed my mind that we might want to look for it, so that we can remove "stray" resources (someone created a CredentialRequest on a non-STS cluster breaking something). But first: we currently have no delete-code for this type of resource and second I honestly don't know how this should happen other than someone restoring a backup from a STS cluster to a non-STS cluster. :shrug: